manylinux-build.sh

```diff
@@ -3,10 +3,10 @@
 set -e -x -o pipefail
 
 
-dockcross_version=20240404-2c6c0a5
```

Comment:
I fixed version for reproducible builds needs to be retained.

Reply:
I will check.
For me was working
![IMG_2781](https://github.com/user-attachments/assets/02593144-5b05-4c24-b5b0-737d0a52b3f1)